e2e/cypress/integration/channel_sidebar/DM_category_spec.js

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @DM category
```

Comment:
Please remove `@prod`. We'll add it once we found this spec to be stable for prod test.  For group metadata, it should be connected by `_`.
```suggestion
// Group: @dm_category
```